lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -280,7 +287,9 @@ def main():
             password=dict(no_log=True),
             group=dict(),
             email=dict(),
-            state=dict(choices=['absent', 'present'], default='present')
+            state=dict(choices=['absent', 'present'], default='present'),
+            update_password=dict(choices=['always', 'on_create'],
+                                 default='always')
         ),
```

Comment:
Can you add trailing comma to avoid future noise?
(Ignore if ansible's lint disagrees)